avm/res/network/private-dns-zone/tests/e2e/defaults/main.test.bicep

```diff
@@ -0,0 +1,61 @@
+targetScope = 'subscription'
+
+metadata name = 'Using only defaults'
+metadata description = '''
+This instance deploys the module with the minimum set of required parameters.
+> **Note:** The test currently implements additional non-required parameters to cater for a test-specific limitation.
+'''
+
+// ========== //
+// Parameters //
+// ========== //
+
+@description('Optional. The name of the resource group to deploy for testing purposes.')
+@maxLength(90)
+param resourceGroupName string = 'dep-${namePrefix}-network.privatednszones-${serviceShort}-rg'
+
+@description('Optional. The location to deploy resources to.')
+param location string = deployment().location
+
+@description('Optional. A short identifier for the kind of deployment. Should be kept short to not run into resource-name length-constraints.')
+param serviceShort string = 'npdzmin'
+
+@description('Optional. A token to inject into the name of each resource.')
+param namePrefix string = '#_namePrefix_#'
+
+// ============ //
+// Dependencies //
+// ============ //
+
+// General resources
+// =================
+resource resourceGroup 'Microsoft.Resources/resourceGroups@2021-04-01' = {
+  name: resourceGroupName
+  location: location
+}
+
+// ============== //
+// Test Execution //
+// ============== //
+
+@batchSize(1)
+module testDeployment '../../../main.bicep' = [for iteration in [ 'init', 'idem' ]: {
+  scope: resourceGroup
+  name: '${uniqueString(deployment().name, location)}-test-${serviceShort}-${iteration}'
+  params: {
+    name: '${namePrefix}${serviceShort}001.com'
```

Comment:
I sense this is causing warnings. As we don't have the exclusion rule anymore, please pass the location through: `location: location` for this and the other test files

Reply:
In the other test files we use `location: 'global` as a private dns zone is a non regional resource we can't update the location param in the test file otherwise the RG would be in the wrong region?

Reply:
Right, but we should be able to pass global explicitely. Just asking as usually bicep linter complains when using default values for location

Reply:
completed